src/editor.py

```diff
@@ -216,7 +216,8 @@ def create_text(self, atPos = 0, lasts = 1, text = "", anchor = CENTER, align =
 				"text": text,
 				"anchor": anchor,
 				"align": align,
-				"offset": [0,0]
+				"offset": [0,0],
+				"key": None,
```

Comment:
To be fair I think having an unused "key" property might be a bad idea (as it'd just add more data to a chart and end up never being used), maybe changing the code that handles duplicate might be a better idea

Reply:
I've changed the "d" key to only duplicate notes